Gemfile

```diff
@@ -16,7 +16,9 @@ gem 'squeel'
 gem 'quiet_assets'
 
 gem 'openstax_utilities'
-gem 'openstax_api'
+# gem 'openstax_api', '6.1.3'
+gem 'openstax_api', github: 'openstax/openstax_api', branch: 'fix/rails-3.2'
+
```

Comment:
**NOTE!** Remove this and uncomment versioned gem once it's PR is merged and it's released.